assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/shared/rate-price.tsx

```diff
@@ -16,7 +16,7 @@ export const RatePrice = ( {
 }: {
 	minRate: CartShippingPackageShippingRate | undefined;
 	maxRate: CartShippingPackageShippingRate | undefined;
-	multiple: boolean;
+	multiple?: boolean;
```

Comment:
This change will fix the TypeScript error in `assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/block.tsx` for not passing the `multiple` prop.